Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -15,6 +19,11 @@ public partial class Video : BaseComponent, IAsyncDisposable
 {
     #region Methods
 
+    ///<summary>
+    /// ID of youtube video link.
+    /// </summary>
+    public string YoutubeVideoID;
```

Comment:
Should this have the setter as public?  Seems like we set this ourselves and should not allow anyone to override it?

Reply:
And should we really expose something specific to `Youtube` like this? Which we don't seem to require to do anything with it, and are adding specific regex logic that could perhaps change with time? i.e youtube changes the way they do ids or something?